components/Card.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export default function card({ voc }) {
+  return (
+    <>
+      <StyledSection>
+        <p>{voc.word}</p>
+      </StyledSection>
+    </>
+  );
+}
+
+const StyledSection = styled.section`
```

Comment:
Why did you choose a `<section>` and not a `<li>` element? I would have thought that since the `cardlist` component is an `<ul>`, the cards rendered in it would be `<li>` elements.